lib/dndstrings-captions.sty

```diff
@@ -196,57 +196,57 @@
+\addto\captionsngerman{
+    \def\armorclassname{Rüstungsklasse}
+    \def\hitpointsname{Trefferpunkte}
+    \def\speedname{Bewegungsrate}
+    \def\strstatname{STR}
+    \def\dexstatname{GES}
+    \def\constatname{KON}
+    \def\intstatname{INT}
+    \def\wisstatname{WEI}
+    \def\chastatname{CHA}
+    \def\skillsname{Fertigkeiten}
+    \def\dimmname{Schadensimmunitäten}
+    \def\dvulname{Schadensanfälligkeiten}
+    \def\dresname{Schadensresistenzen}
+    \def\cimmname{Zustandsimmunitäten}
+    \def\savesname{Rettungswürfe}
+    \def\sensesname{Sinne}
+    \def\languagesname{Sprachen}
+    \def\challengename{Schwierigkeitseinstufung}
     \newcommand\xpname{EP}
     % Monster attack
-    \renewcommand\ftname{Fuß}
-    \renewcommand\meleeattackname{Nahkampfangriff}
-    \renewcommand\rangedattackname{Fernkampangriff}
-    \renewcommand\meleeorrangedattackname{Nah-~oder~Fernkampfangriff}
-    \renewcommand\tohitname{um~zu~treffen}
+    \renewcommand\ftname{m}
+    \renewcommand\meleeattackname{Nahkampf-Waffenangriff}
+    \renewcommand\rangedattackname{Fernkampf-Waffenangriff}
+    \renewcommand\meleeorrangedattackname{Nah-~oder~Fernkampf-Waffenangriff}
+    \renewcommand\tohitname{zum~Treffen}
     \renewcommand\reachname{Reichweite}
-    \renewcommand\rangename{Bereich}
+    \renewcommand\rangename{Reichweite}
     \renewcommand\hitname{Treffer}
     \renewcommand\damagename{Schaden}
     % Spell levels
     \renewcommand\spellcantripsname{Zaubertricks}
-    \renewcommand\spellfirstlevelname{1.~Stufe}
-    \renewcommand\spellsecondlevelname{2.~Stufe}
-    \renewcommand\spellthirdlevelname{3.~Stufe}
-    \renewcommand\spellfourthlevelname{4.~Stufe}
-    \renewcommand\spellfifthlevelname{5.~Stufe}
-    \renewcommand\spellsixthlevelname{6.~Stufe}
-    \renewcommand\spellseventhlevelname{7.~Stufe}
-    \renewcommand\spelleighthlevelname{8.~Stufe}
-    \renewcommand\spellninthlevelname{9.~Stufe}
+    \renewcommand\spellfirstlevelname{1.~Grad}
+    \renewcommand\spellsecondlevelname{2.~Grad}
+    \renewcommand\spellthirdlevelname{3.~Grad}
+    \renewcommand\spellfourthlevelname{4.~Grad}
+    \renewcommand\spellfifthlevelname{5.~Grad}
+    \renewcommand\spellsixthlevelname{6.~Grad}
+    \renewcommand\spellseventhlevelname{7.~Grad}
+    \renewcommand\spelleighthlevelname{8.~Grad}
+    \renewcommand\spellninthlevelname{9.~Grad}
     % Spell slots
-    \renewcommand\spellatwillname{nach~belieben}
-    \renewcommand\spelloneslotname{1~Slot}
-    \renewcommand\spelltwoslotsname{2~Slots}
-    \renewcommand\spellthreeslotsname{3~Slots}
-    \renewcommand\spellfourslotsname{4~Slots}
+    \renewcommand\spellatwillname{jederzeit}
+    \renewcommand\spelloneslotname{1~Platz}
+    \renewcommand\spelltwoslotsname{2~Plätze}
+    \renewcommand\spellthreeslotsname{3~Plätze}
+    \renewcommand\spellfourslotsname{4~Plätze}
     % Innate spellcasting
-    \renewcommand\innateatwillname{Nach~belieben}
+    \renewcommand\innateatwillname{Jederzeit}
```

Comment:
In the monstermanual it is `Willentlich`

Reply:
Changed it.